pkg/actions/nodegroup/create.go

```diff
@@ -80,7 +81,7 @@ func (m *Manager) Create(options CreateOpts, nodegroupFilter filter.NodegroupFil
 	}
 
 	if !options.DryRun {
-		if err := m.init.Normalize(nodePools, cfg.Metadata); err != nil {
+		if err := m.init.Normalize(context.TODO(), nodePools, cfg.Metadata); err != nil {
```

Comment:
```suggestion
		if err := m.init.Normalize(context.TODO(), nodePools, cfg.Metadata); err != nil {
```

can we pass the ctx down from `ctl/nodegroup/create.go`?

Reply:
Done.